### ISSUE TITLE
Missing indexes in _CYBERWAY_ database

### ISSUE THREAD

**Issue by s-medvedev** [GolosChain/cyberway#98]
@s-medvedev commented on [Thu Nov 08 2018](https://github.com/GolosChain/golos-smart/issues/200)

**Impact:**
Performance degradation

**STR:**
1. Start Cyberway
2. Show indexes in mongo database:
```
use _CYBERWAY_
db.account.getIndexes()
```

**Actual result:**
```
> db.account.getIndexes()
[
	{
		"v" : 2,
		"key" : {
			"_id" : 1
		},
		"name" : "_id_",
		"ns" : "_CYBERWAY_.account"
	}
]
```

**Expected result:**
Indexes with name `id` and `name` are presented

**Additional info:**
May be the reason is #97

**Comment by afalaleev** [comment]
Closed by #105

**Comment by afalaleev** [comment]
#106
